### ISSUE TITLE
Correct way to apply a preset

### ISSUE THREAD

**Issue by DenL** [Bluefissure/HousingPos#9]
I'm not sure if I'm using the plugin correctly. After importing a preset, I would have to
1) Turn on 'Place Anywhere' for BDTH
2) Select one furniture in rotation mode
3) Click 'Set' for that furniture
4) Repeat 2-3 for all furniture.
to apply the preset.

Is there a faster way to apply the entire preset?

**Comment by Bluefissure** [comment]
Yes, you need to manually set each furniture. I don't know whether there's a way to set it through the API without interacting with the game. Does bdth have it? Like, click items in the list and it'll automatically be set? I haven't use these tools for a long time.
